modules/ROOT/pages/ogm/installation.adoc

```diff
@@ -287,11 +304,11 @@ In your application directory, you can run this application:
 node index.js
 ----
 
-Which should output:
+You should see the following output:
 
 [source, bash, indent=0]
 ----
-Example app listening at http://localhost:4000
+Example app listening at http://localhost:4000/users
 ----
 
-The REST API should now be ready to accept requests at the URL logged.
+The REST API should now be ready to accept requests at that URL.
```

Comment:
Arguably the REST API is available at `http://localhost:4000/` and the only working route is `/users`.

I'm not sure it's a necessary distinction, but for the sake of correctness it's adding consideration for.

Reply:
updated